heartbeat/galera

```diff
@@ -68,6 +68,24 @@
 . ${OCF_FUNCTIONS_DIR}/ocf-shellfuncs
 . ${OCF_FUNCTIONS_DIR}/mysql-common.sh
 
+
+# FIXME:  So at the moment, the proof of concept is using SSH to
+# get at the "other" cluster, and to authenticate I just have id_rsa keys
+# copied to all the controllers under "root", since that's who can run
+# pcs commands.  So the RA on a docker container has root access to all
+# the controller nodes.
+#
+# So we would like to either:
+#
+# a. ssh as some other locked down account that can still run the
+# necessary pcs commands on the other cluster? or some setuid script that
+# does the things we need?
+#
+# b. totally other means of invoking pcs commands on remote cluster?  web
+# service or something?
+#
+SSH_CMD="${SSH} -oStrictHostKeyChecking=no"
```

Comment:
can we make this configurable?

Reply:
yeah, what other commands might we be using?